src/nav/nav.css

```
@@ -29,4 +29,132 @@
     display: flex;
     gap: 45px;
     padding-right: 100px;
-}
\ No newline at end of file
+}
+
+.navig_item {
```

what do you do with all the free time that you saved by using `navig` instead of `navigation`?
```suggestion
.navigation_item {
```